p/scripts/main.js

```diff
@@ -692,21 +692,113 @@ function init_posts() {
 	}
 }
 
+//Gets URL search parameter value.
+//Does not use URLSearchParams to ensure coverage of older versions of Internet Explorer.
+function get_search_param_value(param_name) {
```

Comment:
Let's drop IE11 for non-essential features. I suggest you use URLSearchParams instead of this function (testing that URLSearchParams exists otherwise do nothing).

Reply:
Okay,

Reply:
Need one clarification. Shall I remove the entire feature if URLSearchParams is not available or shall I just put this part of code inside an availability condition? I'm not sure if the latter is desirable. Latter would also result in inconsistent behavior across two sets of browsers. Please advise.

Reply:
We will probably drop IE11 in the not too distant future, so I would rather like not to introduce additional non-essential code for IE11, especially for non-default options. I suggest a test of whether `URLSearchParams` exists in the `display_open_categories()` function, and simply exit if it does not exist.

Reply:
Okay.